actions/mistral_create_branch.sh

```diff
@@ -81,7 +81,10 @@ if [[ $(grep -c . <<< "${REQUIREMENTS}") > 1 ]]; then
     echo "Updating requirements.txt..."
     REQUIREMENTS=`echo "${REQUIREMENTS}" | sed '/https:\/\/github.com\/stackstorm/Id'`
     echo "${REQUIREMENTS}" > requirements.txt
-    grep -q 'python-mistralclient' requirements.txt || echo "git+https://github.com/StackStorm/python-mistralclient.git@${BRANCH}#egg=python-mistralclient" >> requirements.txt
+    # Note: python-mistralclient must come before troveclient since troveclient depends on mistralclient
+    # In This case we insert it before python-barbicanclient
+    MISTRALCLIENT_LINE="git+https://github.com/StackStorm/python-mistralclient.git@${BRANCH}#egg=python-mistralclient"
+    grep -q 'python-mistralclient' requirements.txt || sed -i "/python-barbicanclient/ i ${MISTRALCLIENT_LINE}" requirements.txt
```

Comment:
Correct, this could be problematic but we really have nothing else to fall back to.

Not sure it would work correctly if we insert mistral client at the very beginning.